hudi-common/src/main/avro/HoodieCleanMetadata.avsc

```diff
@@ -23,6 +23,7 @@
      {"name": "timeTakenInMillis", "type": "long"},
      {"name": "totalFilesDeleted", "type": "int"},
      {"name": "earliestCommitToRetain", "type": "string"},
+     {"name": "lastCompletedCommitTimestamp", "type": "string", "default" : ""},
```

Comment:
why empty string. we could go w/ null ?

Reply:
As I mentioned earlier, the main reason is to be compatible with the old version. The default value should not be null,
```scala
numCommits = commitTimeline.findInstantsAfter(lastCompletedCommitTimestamp).countInstants()
```
Null and empty return different results